timelock-impl/src/main/java/com/palantir/atlasdb/timelock/adjudicate/Constants.java

```diff
@@ -34,15 +34,17 @@ private Constants() {
     public static final ServiceLevelObjectiveSpecification LEADER_TIME_SERVICE_LEVEL_OBJECTIVES
             = ServiceLevelObjectiveSpecification.builder()
             .maximumPermittedP99(Duration.ofMillis(200))
-            .minimumRequestRateForConsideration(1)
+            .minimumRequestRateForConsideration(0.1)
             .maximumPermittedErrorProportion(0.5)
+            .p99Multiplier(100)
             .build();
 
     public static final ServiceLevelObjectiveSpecification START_TRANSACTION_SERVICE_LEVEL_OBJECTIVES
             = ServiceLevelObjectiveSpecification.builder()
             .maximumPermittedP99(Duration.ofMillis(500))
-            .minimumRequestRateForConsideration(1)
+            .minimumRequestRateForConsideration(0.1)
```

Comment:
Could we make this `0.02`?

The reason is because these endpoints on the AtlasDB side are autobatched. So the current p99 multiplier there means we'll say unhealthy on 50 seconds or above, and it would be good to catch a server that is queueing up 40 second autobatcher calls.